ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -73,8 +147,74 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
     static get is() {
       return 'nuxeo-checkbox-aggregation';
     }
+
+    static get properties() {
+      return {
+        /**
+         * Make the element collapsible
+         */
+        collapsible: {
+          type: Boolean,
+          value: false,
+        },
+
+        /**
+         * Label of the element
+         */
+        label: {
+          type: String,
+          value: '',
+        },
+
+        _showAll: {
+          type: Boolean,
+          value: false,
+          readOnly: true,
+        },
+
+        _visibleBuckets: {
+          type: Array,
+          value: () => [],
+          readOnly: true,
+        },
+
+        /**
+         * If collapsible, the default visible items.
+         */
+        visibleItems: {
+          type: Number,
+          value: 8,
+        },
+      };
+    }
+
+    static get observers() {
+      return ['_setBuckets(buckets, visibleItems)'];
+    }
+
+    _setBuckets() {
+      if (this.buckets && this.buckets.length > 0) {
+        this._set_visibleBuckets(this.buckets.slice(0, this.visibleItems));
+      }
+    }
+
+    _toggleShow() {
+      this._set_showAll(!this._showAll);
+      if (this._showAll) {
+        this._set_visibleBuckets(this.buckets);
```

Comment:
guess the `visibleBuckets` computed property also depends on `_showAll` and `visibleItems`